main.py

```diff
@@ -170,6 +204,10 @@ def main():
             else:
                 raise ValueError("No eos_token or bos_token found")
         tokenizer.pad_token = tokenizer.eos_token
+        # Padding on the right is needed to cut off padding in `complete_code`
+        tokenizer.padding_side = "right"
+        
+        args.generations_path = "generations.json" if args.generations_path is None else args.generations_path
```

Comment:
hahah yea this was hardcoded, I also changed it in this [PR](https://github.com/bigcode-project/bigcode-evaluation-harness/pull/62)